licenses.html

```diff
@@ -44,7 +44,7 @@ <h1>Third Party Licenses</h1>
     
         <h2>Overview of licenses:</h2>
         <ul class="licenses-overview">
-            <li><a href="#Apache-2.0">Apache License 2.0</a> (482)</li>
+            <li><a href="#Apache-2.0">Apache License 2.0</a> (485)</li>
```

Comment:
I assume this happened automatically from some of our tooling? Related to adding the glob feature maybe? Probably want to double check with someone who knows better (maybe @lennyburdette) that there's no issue with this.

Reply:
That was my assumption, but I'll confirm.

Reply:
Confirmed that this is done via `cargo xtask all`, which in turn calls `cargo about` - see [licenses.rs](https://github.com/apollographql/router/blob/dev/xtask/src/commands/licenses.rs).

Reply:
I would remove it from the diff, that xtask isn’t deterministic so it’s just noise

Reply:
If I were to do that, then every time someone runs `cargo xtask all` from now on, they'll get this diff locally... Seems better to just include it? But I'll defer to you @lennyburdette if you think it should be reverted.

Reply:
Good to know, thanks! I wonder if this shouldn't be in `cargo xtask all`, then, and instead in a dedicated command 🤔. Maybe something we can ask the other contributors about.

Reply:
I reverted the `licenses.html` changes